uber/site_sections/dept_admin.py

```diff
@@ -194,39 +196,39 @@ def dept_requests_export(self, out, session, department_id, requested_any=False,
 
     @csv_file
     def overworked_attendees(self, out, session):
```

Comment:
I don't really have a great way to check this for performance issues, but I suspect it may cause issues.